tests/unit/customizations/history/test_db.py

```diff
@@ -617,6 +617,20 @@ def _get_request_id_for_event_type(self, builder, event_type):
         record = builder.build_record(event_type, {'body': b''}, '')
         return record.get('request_id')
 
+    def test_can_process_http_request_with_no_body(self):
+        builder = RecordBuilder()
+        try:
+            builder.build_record('HTTP_REQUEST', {}, '')
```

Comment:
So it is not guaranteed that a body will be available? I think in my code I assume at least the `'body'` key will be present but maybe set as `None`